app/services/postcode_validator.rb

```diff
@@ -0,0 +1,9 @@
+class PostcodeValidator
```

Comment:
Would this validator be best placed in `app/validators`, then we can use `ActiveModel::Validations`'s `validates_with` to invoke?